xrpl/deploy-multisig.js

```diff
@@ -0,0 +1,112 @@
+const xrpl = require('xrpl');
+const { Command, Option } = require('commander');
+const { mainProcessor, getWallet, getAccountInfo, sendTransaction, hex } = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const { printInfo, printWarn } = require('../common');
+
+const MAX_TICKET_COUNT = 250;
+const KEY_TYPE = xrpl.ECDSA.secp256k1;
+
+const BASE_RESERVE_XRP = 10;
+const OWNER_RESERVE_XRP = 0.2;
+
+async function processCommand(_, chain, options) {
+    const client = new xrpl.Client(chain.rpc);
```

Comment:
Pass the client in as an arg. The caller can handle connect/disconnect then

Reply:
Addressed in https://github.com/axelarnetwork/axelar-contract-deployments/pull/564/commits/dedd8d8f5f8df582f2507b27f854197d49df4c9f. The client is constructed and connected/disconnected in `mainProcess`.